eth-connector/src/fee.rs

```diff
@@ -0,0 +1,28 @@
+use near_sdk::{
+    borsh::{self, BorshDeserialize, BorshSerialize},
+    collections::UnorderedMap,
+    json_types::U128,
+    serde::{Deserialize, Serialize},
+    AccountId,
+};
+
+#[derive(Copy, Debug, Serialize, Deserialize, Clone)]
+pub enum FeeType {
+    Deposit,
+    Withdraw,
+}
+
+#[derive(BorshDeserialize, BorshSerialize, Debug, Clone, Serialize, Deserialize, Copy)]
+pub struct Fee {
+    pub fee_percentage: U128,
+    pub lower_bound: Option<U128>,
+    pub upper_bound: Option<U128>,
+}
```

Comment:
Please describe all the fields.

Reply:
c4aec919755ea40eaee25e7601687a83cde40552